### ISSUE TITLE
Modify config (LR, per-layer, etc.) of deserialized model w/ updater state

### ISSUE THREAD

**Issue by crockpotveggies** [deeplearning4j/deeplearning4j#2779]
Currently there isn't a straightforward way to resume training of a trained model while changing certain parameters such as learning rate. 
```
network = ModelSerializer.restoreComputationGraph(existingModel, true);
```

As per Gitter discussion, this might be useful:
```
network.modifyConfiguration callback?
network.modifyConf( conf -> do stuff return newConf;);
```

**Comment by AlexDBlack** [comment]
This is coming up for transfer learning as well, fyi. So it's something we're starting to think about.

**Comment by mjakobus** [comment]
I'm also interested in such kind of feature. 
My use case is, that I want to change the l1 and l2 hyperparameters during training. Meaning, that I already have an initialized ComputationGraph which is already trained for a few epochs.
Somewhere I read, that it might be possible by initializing a new network with the changed hyperparameters and then transfer the parameters of the old network. It would be great, if these hyperparameters could be changed without reinitialization of the network.

**Comment by crockpotveggies** [comment]
Is there any interest in adding this to the UI? I'm mulling adding some features incuding those mentioned by @mjakobus to the UI for the purpose of stage-wise training, especially for long-term training jobs. @AlexDBlack

**Comment by eraly** [comment]
So we have this with the new transfer learning API. The fine tune configuration will allow you to override any of the learning configs on all the layers in a computation graph or model.

https://github.com/deeplearning4j/dl4j-examples/tree/master/dl4j-examples/src/main/java/org/deeplearning4j/examples/transferlearning/vgg16
